src/loader-components/PropsTable.module.scss

```diff
@@ -1,4 +1,5 @@
 @import '../mdx/MarkdownTags.module';
+@import './clinical-lowlight-theme/PropsTableModule';
```

Comment:
Left a commend below, this import should be `PropsTable.module` after the file rename